### ISSUE TITLE
Seem that the code in example-forms not match those in Youtube video

### ISSUE THREAD

**Issue by adalidda** [VulcanJS/Vulcan-Starter#49]
Hi,

I tested example-forms and the presentation is not like in youtube video.
Example: the adressses field is with one edit box and the + before submit is missing.

References: 
https://blog.vulcanjs.org/vulcan-js-1-9-nested-forms-support-better-image-upload-and-more-8da2076bb43f
https://github.com/VulcanJS/Vulcan-Starter.git
https://www.youtube.com/watch?time_continue=196&v=v3ftplSZWrs

Thank You
Adalidda

**Comment by SachaG** [comment]
This is what I see when I run the example:

![https://d3vv6lp55qjaqc.cloudfront.net/items/2W2N313H2q3s150k1Z3C/Screen%20Shot%202018-04-22%20at%209.29.28.png?X-CloudApp-Visitor-Id=43642&v=66da0e74](https://d3vv6lp55qjaqc.cloudfront.net/items/2W2N313H2q3s150k1Z3C/Screen%20Shot%202018-04-22%20at%209.29.28.png?X-CloudApp-Visitor-Id=43642&v=66da0e74)

Are you seeing something different?

**Comment by adalidda** [comment]
Yes, the addresses is with one blank line and there are nothing before submit.

I download the Vulcan Starter this morning.

Thank You
Adalidda

**Comment by SachaG** [comment]
Maybe you need to update Vulcan itself? Do you have 1.9.1?

**Comment by adalidda** [comment]
I used the npm install that come with Vulcan-Starter. 

OK, I will try again by installing Vulcan 1.9.1 with the example-forms package from the Vulcan-Starter.

Will update you.

Thank You
Adalidda

**Comment by adalidda** [comment]
Dear @SachaG,

I confirmed that:
by git clone https://github.com/VulcanJS/Vulcan.git (version 1.9.1)
and copy the folder example-forms from Vulcan-Starter and meteor add example-forms

Everything is working fine like in the video https://www.youtube.com/watch?time_continue=196&v=v3ftplSZWrs

So it's mean that you just need to upgrade Vulcan-Starter to Vulcan 1.9.1

Thank You so much and the new features in Vulcan 1.9.1 are really a great move.

Adalidda
